autosubmit/platforms/platform.py

```diff
@@ -532,41 +507,6 @@ def add_parameters(self, as_conf):
 
         as_conf.experiment_data['{0}LOGDIR'.format(prefix)] = self.get_files_path()
 
-    def send_file(self, filename, check=True):
-        """
-        Sends a local file to the platform
-        :param check:
-        :param filename: name of the file to send
-        :type filename: str
-        """
-        raise NotImplementedError
-
-    def move_file(self, src, dest):
-        """
-        Moves a file on the platform
-        :param src: source name
-        :type src: str
-        :param dest: destination name
-        :type dest: str
-        """
-        raise NotImplementedError
-
-    def get_file(self, filename, must_exist=True, relative_path='', ignore_log=False, wrapper_failed=False):
-        """
-        Copies a file from the current platform to experiment's tmp folder
-
-        :param wrapper_failed:
-        :param ignore_log:
-        :param filename: file name
-        :type filename: str
-        :param must_exist: If True, raises an exception if file can not be copied
-        :type must_exist: bool
-        :param relative_path: relative path inside tmp folder
-        :type relative_path: str
-        :return: True if file is copied successfully, false otherwise
-        :rtype: bool
-        """
-        raise NotImplementedError
```

Comment:
I think these need to stay? The idea was to provide a default impl that classes implementing this would need to include, or the error would be raised?